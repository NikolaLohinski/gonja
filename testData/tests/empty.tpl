"": {{ "" is empty }}
"nope": {{ "nope" is empty }}
[]: {{ [] is empty }}
["one"]: {{ ["one"] is empty }}
{}: {{ {} is empty }}
{"a": "b"}: {{ {"a": "b"} is empty }}
