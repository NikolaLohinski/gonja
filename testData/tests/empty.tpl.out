"": True
"nope": False
[]: True
["one"]: False
{}: True
{"a": "b"}: False
